### PR TITLE
update node version to 16.13.2

### LINKED ISSUE #8: Update Node version
I did create-react-app with Node version 12s.
Therefore, I'll update to the latest stable version.

### PR DESCRIPTION
**Pull request by h-saitooo**
Add a file to specify the version of nodenv
close #8

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+14.17.6
```
Comment (yukinoda) [inline .node-version:1]: you can use the `engines` field to specify the node version in react projects
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines

**Review by yukinoda — COMMENTED** [review]
commented

File: .node-version (hunk 1 above)
Comment (yukinoda) [reply .node-version:1]: specifying with a `.node-version` file looks fine as well, although I have never used it...
https://stackoverflow.com/questions/27425852/what-uses-respects-the-node-version-file

**Review by yukinoda — APPROVED** [review]

**Comment by h-saitooo** [comment]
@yukinoda 
I added engines setting to package.json
Please re-review it.

**Review by yukinoda — APPROVED** [review]

**Pull request merged**